velox/expression/tests/ExpressionFuzzerTest.cpp

```diff
@@ -60,6 +60,7 @@ int main(int argc, char** argv) {
       "cardinality",
       "element_at",
       "width_bucket",
+      "contains",
```

Comment:
@mbasmanova I need to skip contains to make the expression fuzzer successful as the `contains` may receive input with nested null and throw making the expression fuzzer test fail.
Another option is to add a config item in the `QueryConfig` (named `ignore_nested_null`?) the same solution as I suggested in https://github.com/facebookincubator/velox/pull/7229#issuecomment-1780494249. What do you think?

Reply:
> the contains may receive input with nested null and throw making the expression fuzzer test fail.

This should be fine. Many functions have restrictions on inputs and often Fuzzer generates inputs that violate these restrictions. Failing is OK. Fuzzer should handle it well. Can you dig a bit deeper to see what's happening here?

Reply:
@mbasmanova Oh, I see. It is my mistake to use `VELOX_CHECK`, which throws `VeloxRuntimeError` that is not allowed by the `ExpressionVerifier`. I should use `VELOX_USER_CHECK` that throws `VelxoUserError`. I will fix this.